homeassistant/components/atag/strings.json

```diff
@@ -6,14 +6,14 @@
         "title": "Connect to the device",
         "data": {
           "host": "[%key:common::config_flow::data::host%]",
-          "email": "Email (Optional)",
+          "email": "[%key:common::config_flow::data::email%] (Optional)",
           "port": "[%key:common::config_flow::data::port%]"
         }
       }
     },
     "error": {
       "unauthorized": "Pairing denied, check device for auth request",
-      "connection_error": "Failed to connect, please try again"
+      "connection_error": "[%key:common::config_flow::error::cannot_connect%], please try again"
```

Comment:
You have to replace the full key, not partially

Reply:
Can I remove the remaining part in this case. Since it doesn't add a lot of value in clarifying?

Reply:
Agreed